test/rdoc/test_rdoc_parser_ruby.rb

```diff
@@ -4297,4 +4297,50 @@ class D
     assert_equal 'A::D', a_d.full_name
   end
 
+  BINARY_OPERATOR_METHOD_NAMES =
+    %w[ != !~ % & * ** + - / < << <= <=> == === =~ > >= >> ^ | ~ ]
+
+  def test_coerce_call_seq
```

Comment:
This is still difficult to read. Can you split them into separate test methods?

Reply:
I'd rather not split.  I think there's value in seeing them all at once.  Does the reformatting help?

Reply:
Well I see four tests here:

One for when the operator is handled.

One for methods that aren't handled.

Multiple arguments to operators isn't handled.

Singleton methods aren't handled.

Reply:
Now split into multiple tests.